server/src/main/java/org/apache/iotdb/db/metadata/idtable/IDTable.java

```diff
@@ -194,6 +206,14 @@ static PartialPath translateQueryPath(PartialPath fullPath) {
     }
   }
 
+  /**
+   * get DiskSchemaEntries from disk file
+   *
+   * @param schemaEntries get the disk pointers from schemaEntries
+   * @return DiskSchemaEntries
+   */
+  List<DiskSchemaEntry> getDiskSchemaEntries(List<SchemaEntry> schemaEntries);
+
```

Comment:
It seems this is only used in test. Annotation ```@TestOnly``` could be added.

Reply:
fixed